package.json

```diff
@@ -34,9 +33,6 @@
     "xstream": "*"
   },
   "devDependencies": {
-    "@cycle/run": "3.1.0",
-    "typescript": "^2.3.4",
-    "typings": "^2.1.1",
-    "xstream": "^10.8.0"
```

Comment:
xstream is a devDependency because the type definitions need to be generated. If it is not, the type of streams we use cannot be known. It is definitely needed. I'm not sure if we need the "@cycle/run" dependency anymore, but I'm sure we need "xstream". Maybe updating the version will be enough. "typings" as a dependency we can surely remove, as we've updated typescript.

I'm pretty sure it works on your machine only because you have somehow already `npm install`ed xstream for this package. And the compilation and type definition generation will fail without xstream, that is why it is definitely a dev dependency.